stylus/include/WhereToFindUs.styl

```diff
@@ -0,0 +1,51 @@
+.WhereToFindUs
+	margin-bottom: 3rem
+	margin-top: 5rem
+	text-align: center
+	>h1
+		font-family: Arvo
+		margin-bottom: 3rem
+	>p
+		margin-right: 33%
+		margin-left: 33%
+	>div:nth-child(3)
+		display: flex
+		flex-wrap: wrap
+		margin-left: 8rem
+		margin-top: 5rem
+		text-align: left
+		>div
+			flex: 1
```

Comment:
I'd actually avoid setting `flex` here. Watch what happens at a high browser width:
![screenshot at 2018-07-09 07 34 22](https://user-images.githubusercontent.com/10747047/42456826-b242af80-834a-11e8-8daf-fec61d4605cc.png)
Setting `flex` allows the items to grow (the 1 here is actually `flex-grow`. `flex` is a shorthand combining a few properties) to take up remaining space, which ends up looking awkward here. Remove it and set `justfiy-content: center` so that a row with fewer items will still be symmetrical.